dynawo/sources/Modeler/DataInterface/DYNLineInterface.h

```diff
@@ -185,6 +187,65 @@ class LineInterface : public ComponentInterface {
    * @copydoc ComponentInterface::exportStateVariablesUnitComponent()
    */
   virtual void exportStateVariablesUnitComponent() = 0;
+
+  /**
+   * @brief Retrieve the permanent limit
+   *
+   * @param season the season to apply
+   * @param number the current limit number, between 1 and 3
```

Comment:
How come there are 3 sides on a Line? Moreover we should use an enum and not a free number

Reply:
Using an enum for a number seemed to me odd. As for the 3 sides, I don't know why but this is the way it is supported in powsybl

Reply:
It is supported for 3winding transofmers, not lines. Moreover they do use an enum and we should do the same, similarly to what is done in the dynamic models

Reply:
OK. I'll keep the 3 current limit for consistency with powsybl capabilities